backend/src/triage/dto/update-alert.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { Priority, AlertType } from '@prisma/client';
+import { IsOptional, IsEnum, IsNumber } from 'class-validator';
+
+export class UpdateAlertDto {
+  @IsOptional()
+  @IsNumber()
+  confidence_per?: number;
+
+  @IsOptional()
+  @IsEnum(Priority)
+  priority?: Priority;
+
+  @IsOptional()
+  @IsEnum(AlertType)
+  alertType?: AlertType;
```

Comment:
Addition to the manual update.

```suggestion
  alertType?: AlertType;
  
  @IsOptional()
  @IsEnum(AlertMatrix)
  alert_matrix?: AlertMatrix;
```